mutiny-core/src/nostr/mod.rs

```diff
@@ -183,6 +183,34 @@ impl<S: MutinyStorage> NostrManager<S> {
         Ok(nwc_profile)
     }
 
+    pub fn set_nwc_profile_require_approval(
```

Comment:
these are helpers? I believe there's a generic method that you can override any profile IIRC from a conversation with @benthecarman